src/modules/import.js

```diff
@@ -250,8 +280,34 @@ export const startImport = () => {
           fileType
         });
         dispatch(onFinished(dest.docsWritten));
-        dispatch(appRegistryEmit('import-finished', size, fileType));
-        dispatch(globalAppRegistryEmit('import-finished', size, fileType));
+        dispatch(appRegistryEmit('import-finished',
+          size,
+          fileType,
+          dest.docsWritten,
+          exclude.length > 0,
+          transform.length > 0
+        ));
+        /**
+         * TODO: lucas: For metrics:
+         *
+         * "resource": "Import",
+         * "action": "completed",
+         * "user_id": "bce2e94b-7e2f-463d-9555-4a6586322cc4",
```

Comment:
i think [`user_id`](https://github.com/mongodb-js/compass-metrics/blob/master/src/modules/setup.js#L95), [`created_at`](https://github.com/mongodb-js/compass-metrics/blob/master/src/modules/setup.js#L96), [`compass_version`](https://github.com/mongodb-js/compass-metrics/blob/master/src/modules/rules.js#L29), [`resource`](https://github.com/mongodb-js/compass-metrics/blob/master/src/modules/rules.js#L301), [`action`](https://github.com/mongodb-js/compass-metrics/blob/master/src/modules/rules.js#L302) are all fields that get appended by the metrics module itself, so you shouldn't have to worry about it here!